src/scss/footer.scss

```diff
@@ -0,0 +1,96 @@
+#footer {
+	display: block;
+	min-height: 0 !important;
+	color: #ffffff;
+}
+
+.newsletter {
+	width: 100%;
+	background-color: #222222;
+	padding-top: 100px;
+	padding-bottom: 100px;
+	text-align: center;
+
+	p {
+		text-transform: uppercase;
+		font-family: 'Museo Sans', 'Lato', sans-serif;
+		font-size: 14px;
+		padding-right: 10px;
+		letter-spacing: 1px;
+	}
+
+	input {
+			background-color: transparent;
+			font-size: $buttons-font-size;
+			color: $opfront-white;
+			outline: none;
+			padding: 8px 14px;
+			max-width: 250px;
+			border: 2px solid #4A4A4A;
+			text-transform: none;
+	}
+}
+
+.sitemap {
+	position: absolute;
+	width: 100%;
+	background-color: #000000;
+
+	ul {
+		padding: 20px 0 20px 0;
+	}
+
+	li {
+		display: inline;
+		padding-right: 20px;
+		font-family: 'Museo Sans', 'Lato', sans-serif;
+		font-size: 14px;
+	}
+
+	a {
+		color: $opfront-white;
+	}
+
+	.social-media {
+		float: right;
+
+		li {
+			padding: 8px;
+		}
+
+		a {
+			color: $opfront-white;
+		}
+	}
+}
+
+.inline {
+	display: inline-block;
```

Comment:
I see a bad smell here :P. It is possible that you need to force inline display on certain elements, but I doubt the complexity of the footer needs that. I'll continue reviewing and i'll come back later.